www/playlist

```diff
@@ -234,6 +241,17 @@ if ($errMsg) {
             $hasart = $rec['hasart'];
             $pagevsn = $rec['pagevsn'];
             $onPlayList = ($listtype == "reviewideas" && $rec['onPlayList']);
+            $user_time = "";
+            if ($rec['time_in_minutes'] && $rec['time_id']) {
+                if ($rec['time_id'] > 388) {      
+                    // If the time is public
+                    $user_time = convertTimeToText($rec['time_in_minutes']);
+                } else if ($uid == $curuser) {
+                    // If the time is anonymous, and belongs to the current user
+                    $user_time = convertTimeToText($rec['time_in_minutes']);
+                    $user_time .= " (anonymous <a href='https://ifdb.org/news?item=152'>until December 19</a>)";
```

Comment:
Rather than add another place where we're checking the time ID, I think I want to wait on merging this PR until after Dec 19. That way, we won't have to remember to rip it out again in a few weeks.

Reply:
Ok.